libs/blocks/marketo/marketo.js

```diff
@@ -198,6 +221,15 @@ export default function init(el) {
     return;
   }
 
+  const searchParams = new URLSearchParams(window.location.search);
+  const ungated = searchParams.get(FORM_PARAM) === 'off';
+
+  if (ungated) {
```

Comment:
We should check if we are on a one page
```suggestion
  if (formData.[SUCCESS_TYPE] === 'section' && ungated) {
```